source/content.ts

```diff
@@ -59,6 +59,7 @@ import './features/linkify-issues-in-titles';
 import './features/embed-gist-inline';
 import './features/extend-status-labels';
 import './features/highlight-closing-prs-in-open-issues';
+import './features/highlight-user-prs';
```

Comment:
Maybe we can rename this to `highlight-own-prs` to match the wording of `hide-own-stars`